src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,53 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private StorageImpl<K, V>[] storage = new StorageImpl[10];
+    private K key;
+    private V value;
+    private byte size = 0;
```

Comment:
```suggestion
    private byte size;
```

it will be initialized by 0 by default